docker-for-windows/release-notes.md

```diff
@@ -16,10 +16,34 @@ This page contains information about the new features, improvements, known issue
 > Starting with Docker Desktop 3.0.0, Stable and Edge releases are combined into a single release stream for all users. Updates to Docker Desktop will now be available automatically as delta updates from the previous version. This means, when there is a newer version of Docker Desktop, it will be automatically downloaded to your machine. All you need to do is to click **Update and restart** from the Docker menu to install the latest update.
 {: .important }
 
+## Docker Desktop 3.3.0
+2021-04-08
+
+> [Download](https://desktop.docker.com/win/stable/amd64/Docker%20Desktop%20Installer.exe)
+
+### Bug fixes and minor changes
+
+- Fixed an issue related to docker-compose action https://github.com/docker/for-win/issues/10564
+- Ensure `--add-host host.docker.internal:host-gateway` causes `host.docker.internal` resolves to the host IP, rather than the IP of the IP router. See [docker/for-linux#264](https://github.com/docker/for-linux/issues/264).
+- Fixes port allocation for Windows containers: https://github.com/docker/for-win/issues/10552
+- Fixed an issue where the dashboard opened the container in browser with port 0
+- Perform a filesystem check when starting the Linux VM
+- Fixed pulling an image from Docker Hub from the dashboard
```

Comment:
Does it make sense to clarify it's the Desktop dashboard as we also have the Hub Image Management dashboard? 

'Fixed an issue when pulling an image from Docker Hub using the Docker Desktop dashboard.'

Reply:
Could you also update the Mac release notes accordingly?

Reply:
Sure, thanks!